Administrator Guide/NFS-Ganesha GlusterFS Integration.md

```diff
@@ -24,7 +24,7 @@ NFS-Ganesha is a user space file server for the NFS protocol with support for NF
 where:
 /var/log/ganesha.log is the default log file for the ganesha process.
 /etc/ganesha/ganesha.conf is the default configuration file
-NIV_EVENT is the default log level.
+NIV_EVENT is the default log level. You can modify all the options in /run/sysconfig/ganesha and restart nfs-ganesha
```

Comment:
This is rendered in a box in markdown. The `You can modify` line has to move out of that box.